web/src/main/java/org/apache/shiro/web/servlet/SimpleCookie.java

```diff
@@ -215,7 +215,7 @@ public void saveTo(HttpServletRequest request, HttpServletResponse response) {
         addCookieHeader(response, name, value, comment, domain, path, maxAge, version, secure, httpOnly);
     }
 
-    private void addCookieHeader(HttpServletResponse response, String name, String value, String comment,
+    protected void addCookieHeader(HttpServletResponse response, String name, String value, String comment,
```

Comment:
Instead of changing this method signature, the same thing could probably be done buy overriding `buildHeaderValue()`